internal/coremsgs/en_api_translations.go

```diff
@@ -16,190 +16,193 @@
 
 package coremsgs
 
-import "github.com/hyperledger/firefly-common/pkg/i18n"
+import (
+	"github.com/hyperledger/firefly-common/pkg/i18n"
+	"golang.org/x/text/language"
+)
 
 var ffm = i18n.FFM
 
 //revive:disable
 var (
-	CoreSystemNSDescription = ffm("core.systemNSDescription", "FireFly system namespace")
+	CoreSystemNSDescription = ffm(language.AmericanEnglish, "core.systemNSDescription", "FireFly system namespace")
```

Comment:
Can we follow the same pattern from `firefly-common` to make the local `ffm` helper just add the language tag instead of including it on every line?